lua/toggletasks/task.lua

```diff
@@ -172,6 +173,9 @@ function Task:_expand(str, win, opts)
         win_cwd = dirs.win,
         tab_cwd = dirs.tab,
         global_cwd = dirs.global,
+        -- Expand line info
+        file_line_row = line.row,
+        file_line_column = line.column,
```

Comment:
I think it will make more sense to use `cursor_line` and `cursor_column` instead.

Reply:
Already changed